haystack/utils.py

```diff
@@ -49,6 +82,27 @@ def print_answers(results: dict, details: str = "all"):
         pp.pprint(results)
 
 
+def print_documents(results: dict, max_text_len: int=None):
```

Comment:
Makes sense. Eventually, we should also have nicer functions for the Document class itself so that you can just call `print(Document)`, but for now this is fine ...

Reply:
yeah I agree with that!